testing/ef-tests/src/macros/sanity_blocks.rs

```diff
@@ -0,0 +1,88 @@
+#[macro_export]
+macro_rules! test_sanity_blocks {
+    ($processing_fn:path) => {
+        paste::paste! {
+            #[cfg(test)]
+            #[allow(non_snake_case)]
+            mod test_sanity_blocks {
+                use super::*;
+                use ream_consensus::execution_engine::mock_engine::MockExecutionEngine;
+                use std::{fs, path::{Path, PathBuf}, sync::Arc};
+                use tokio::sync::Mutex;
```

Comment:
```suggestion
                use std::{fs, path::Path};
```

these imports aren't used so they can be cleaned up